src/Tribe/PUE/Checker.php

```diff
@@ -428,7 +428,7 @@ public function ajax_validate_key() {
 				} else {
 					$default_success_msg    = sprintf( __( 'Valid Key! Expires on %s', 'tribe-events-calendar' ), $expiration );
 					$response['status']     = isset( $pluginInfo->api_message ) ? 2 : 1;
-					$response['message']    = isset( $pluginInfo->api_message ) ? $pluginInfo->api_message : $default_success_msg;
+					$response['message']    = isset( $pluginInfo->api_message ) ? wp_kses( $pluginInfo->api_message ) : $default_success_msg;
```

Comment:
`wp_kses` second param is required, you need to pass an `Array` of the Elements that are allowed. To remove all HTML `wp_kses( $data , 'strip' );`